dispatch/static/manager/src/js/pages/Images/ImagesIndexPage.js

```diff
@@ -82,8 +83,8 @@ class ImagesPageComponent extends React.Component {
     this.props.clearSelectedImages()
   }
 
-  handleSearchImages(author, tags, query) {
-    this.props.searchImages(author, tags, query)
+  handleSearchImages(query) {
+    this.props.searchImages(null, null, query)
```

Comment:
This implementation is good, storing everything in query is better than it was. We should probably change the definition of imageActions.search to just be search(query) so that we can remove these nulls

Reply:
NVM

The 'nulls' should get the current query.author and query.tags